flytekit/core/type_engine.py

```diff
@@ -129,6 +130,8 @@ def __init__(self, name: str, t: Type[T], enable_type_assertions: bool = True):
         self._t = t
         self._name = name
         self._type_assertions_enabled = enable_type_assertions
+        self._msgpack_encoder: Dict[Type, MessagePackEncoder] = {}
```

Comment:
The encoder is for the follow up PR, which will support `untyped dict`

Reply:
{} not good, delete